string name
float32[] data
float32 vel
geometry_msgs/Pose pose
nav_msgs/Path path